From c423935181a9a92beadb247747f0bcab2c597e20 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Wed, 12 Jun 2013 11:42:34 -0700
Subject: zgrep: usage should say which grep options are not supported

* zgrep.in (usage): Document which grep options are not supported.
Problem reported by Liron Paryente in
<http://lists.gnu.org/archive/html/bug-grep/2013-06/msg00005.html>.
---
 zgrep.in | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/zgrep.in b/zgrep.in
index 154d17f..c149577 100644
--- a/zgrep.in
+++ b/zgrep.in
@@ -42,7 +42,10 @@ usage="Usage: $0 [OPTION]... [-e] PATTERN [FILE]...
 Look for instances of PATTERN in the input FILEs, using their
 uncompressed contents if they are compressed.
 
-OPTIONs are the same as for 'grep'.
+OPTIONs are the same as for 'grep', except that the following 'grep'
+options are not supported: --dereference-recursive (-R), --directories (-d),
+--exclude, --exclude-from, --exclude-dir, --include, --null (-Z),
+--null-data (-z), and --recursive (-r).
 
 Report bugs to <bug-gzip@gnu.org>."
 
-- 
cgit v1.0-41-gc330

